doc/source/catalog_generation.rst

```diff
@@ -359,6 +359,84 @@ ordering in the .ecsv file as well):
     * CI: Concentration index (MagAp1 – MagAp2), in AB magnitude
     * Flags: See Section 2.3.2 for flag value definitions
 
+3.3 Rejection of Cosmic-Ray Dominated Catalogs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Not all sets of observations contain multiple overlapping exposures in the same filter. This makes it impossible
+to ignore all cosmic-rays that have impacted those single exposures.  The contributions of cosmic-rays often
+overwhelm any catalog generated from those single exposures making recognizing astronomical sources almost
+impossible amongst the noise of all the cosmic-rays.  As a result, those catalogs can not be trusted.  In an
+effort to only publish catalogs which provide the highest science value, criteria developed by the Hubble Legacy
+Archive (HLA) has been implemented to recognize those catalogs dominated by cosmic-rays and not provided as an
+output product.
+
+.. note ::
+  This rejection criteria is NOT applied to WFC3/IR or ACS/SBC data since they are not affected by cosmic-rays
+  in the same way as the other detectors.
+
+3.3.1 Rejection Criteria
+"""""""""""""""""""""""""
```

Comment:
I am having trouble deciding if the subsections should be in this order (3.3.1. vs 3.3.2) or in the reverse order.  I wonder if describing the algorithm first, then followed by the nitty-gritty details would be better.

Let us discuss.

Reply:
Swapping the order of the 2 sections does make a bit more sense.  Done in 4bd2200.